fri/src/prover/mod.rs

```diff
@@ -181,9 +186,11 @@ where
             }
         }
 
+        self.build_remainder(channel, &mut evaluations);
+
         // make sure remainder length does not exceed max allowed value
-        let last_layer = &self.layers[self.layers.len() - 1];
-        let remainder_size = last_layer.evaluations.len();
+        let remainder = &self.remainder;
+        let remainder_size = remainder.0.len();
```

Comment:
nit: let's merge this into a single line as:
```Rust
let remainder_size = self.remainder.0.len();
```

Reply:
Done!